src/grandorgue/dialogs/midi-event/GOMidiEventRecvTab.h

```diff
@@ -38,6 +38,7 @@ class GOMidiEventRecvTab : public wxPanel,
 
   GOMidiReceiverBase *m_original;
   GOMidiReceiverEventPatternList m_midi;
+  GOMidiReceiverType m_ReceiverType;
```

Comment:
I'm somewhat concerned about this. I understand the supposed "simplification", but GOMidiReceiverType is a typedef:ed enum that's buried in another header file (src/core/midi/GOMidiReceiverEventPatternList.h) that all breath of old c style coding that should be structured in a better way, in my opinion. To me, this "clarification" is not really so "clear" as there is no class or separate file that can be easily search for to find the defined type.

Reply:
You are right. But extracting GOMidiReceiverType is a subject of a separate pr #1664.